test/junit/scala/collection/mutable/HashMapTest.scala

```diff
@@ -113,19 +113,144 @@ class HashMapTest {
   @Test
   def testUpdateWith(): Unit = {
     val insertIfAbsent: Option[String] => Option[String] = _.orElse(Some("b"))
+
     val hashMap1 = mutable.HashMap(1 -> "a")
     assertEquals(hashMap1.updateWith(1)(insertIfAbsent), Some("a"))
```

Comment:
All `assertEquals` calls have the arguments reversed. (I see this so often in existing tests that I never trust the error messages. JUnit should have done the API the other way around)